packages/frontend/app/root.tsx

```diff
@@ -30,19 +36,39 @@ export const meta: MetaFunction = () => [
 ]
 
 export const loader = async ({ request }: LoaderFunctionArgs) => {
-  const session = await messageStorage.getSession(request.headers.get('cookie'))
+  const cookies = request.headers.get('cookie')
+  await redirectIfUnauthorizedAccess(request.url, cookies)
+
+  let logoutUrl = ''
+  const loggedIn = await isLoggedIn(cookies)
+  if (loggedIn) {
+    const response = await axios.get(
+      `${variables.kratosContainerPublicUrl}/self-service/logout/browser`,
+      {
+        headers: {
+          cookie: cookies
+        },
+        withCredentials: true
+      }
+    )
+    logoutUrl = response.data.logout_url
+    if (!logoutUrl) {
+      logoutUrl = '/auth/manual-logout'
+    }
```

Comment:
I think we should probably log that there was some issue here. 

Also, related, I think we should probably use pino to log here instead (I see some console.logs elsewhere). judging by this recent audit I think we don't want to introduce console.logs here thinkhttps://github.com/interledger/rafiki/issues/2580

Reply:
Does this resolve the logging issue: https://github.com/interledger/rafiki/commit/f1a25c4b56f6a502b976a9de7cd124e0f4293376

Reply:
yes looks good